sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -2313,6 +2313,168 @@ scatter_rgba(AccessorT acc, simd<uint32_t, N> offsets,
                                        __ESIMD_DNS::localAccessorToOffset(acc),
                                    vals, mask);
 }
+
+/// @addtogroup sycl_esimd_raw_send
+/// @{
+
+/// Raw sends. "s" suffix designates "split" variant - i.e. two sources.
```

Comment:
We promised to some of our users that we would add an additional comment telling that raw_send API is extremely low level, similar to assembler and thus is not recommended for usage as it may be tricky to use it correctly and it makes the code less or non-portable, etc.